src/services/audius-backend/TrackDownload.js

```diff
@@ -16,13 +16,13 @@ class TrackDownload {
     )
   }
 
-  static async downloadTrackMobile(cid, creatorNodeGateways, filename, title) {
+  static async downloadTrackMobile(cid, creatorNodeGateways, filename) {
     const urls = creatorNodeGateways.map(
       gateway => new URL(`${gateway}${cid}?filename=${filename}`)
     )
 
     const message = new DownloadTrackMessage({
-      title: title,
+      title: filename.split('.').slice(0, -1).join(''),
```

Comment:
lgtm. i do agree that just passing filename all the way through would be better -- and requires the same amount of release. ok w/ merging this first -- this won't live forever as eventually the UI for downloaded stems should trigger via the react native track page.